mono/eglib/glib.h

```diff
@@ -797,13 +797,17 @@ gunichar  *g_utf16_to_ucs4 (const gunichar2 *str, glong len, glong *items_read,
 gchar     *g_ucs4_to_utf8  (const gunichar *str, glong len, glong *items_read, glong *items_written, GError **err);
 gunichar2 *g_ucs4_to_utf16 (const gunichar *str, glong len, glong *items_read, glong *items_written, GError **err);
 
-#define u8to16(str) g_utf8_to_utf16(str, (glong)strlen(str), NULL, NULL, NULL)
+#define g_u16len monoeg_g_u16len
```

Comment:
This needs to be moved to `eglib-remap.h`

It's not the first time I request it so at least try to follow the existing convention. You might not agree with them but, again, we are open to discussion, not to disruption 😉